vimeo-networking/src/main/java/com/vimeo/networking/model/Interaction.java

```diff
@@ -72,6 +72,28 @@ public String toString() {
         }
     }
 
+    @UseStag
+    public enum IapStatus {
```

Comment:
Maybe break this out into another file?

Reply:
My original plan was to have this a separate file, but I found that other than SearchType and ErrorCode...  all 32 other enums are in the file in which they are used.  So I kept with that for consistency.